android/src/main/java/com/dylanvann/fastimage/FastImageRequestListener.java

```diff
@@ -23,13 +28,64 @@
         this.key = key;
     }
 
+
+    private static WritableMap mapFromResourceWithPalette(Drawable resource, Palette palette) {
+        WritableNativeMap swatchMap = new WritableNativeMap();
+
+        addHexStringIfSwatchValid(palette.getDominantSwatch(), swatchMap, "dominantSwatch");
+        addHexStringIfSwatchValid(palette.getLightVibrantSwatch(), swatchMap, "lightVibrantSwatch");
+        addHexStringIfSwatchValid(palette.getVibrantSwatch(), swatchMap, "vibrantSwatch");
+        addHexStringIfSwatchValid(palette.getDarkVibrantSwatch(), swatchMap, "darkVibrantSwatch");
```

Comment:
I'm not sure what most of this is trying to achieve, what does "palette support" mean?

Reply:
I think it is about this: https://developer.android.com/reference/android/support/v7/graphics/Palette